release/community-stable/amazonlinux/docker/Dockerfile

```diff
@@ -31,14 +31,17 @@ ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION \
     POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-AmazonLinux-${fromTag}
 
 # Installation
-RUN curl -sSL ${PS_PACKAGE_URL} -o /tmp/powershell-linux.rpm \
+RUN \
+    # download PowerShell rpm package
+    curl -sSL ${PS_PACKAGE_URL} -o /tmp/powershell-linux.rpm \
+    # install EPEL repository
+    && yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
     # install dependencies
     && yum install -y \
 	    # required for help in PowerShell
 	    less \
-	    # NTLM remoting
-	    epel-release \
-            gssntlmssp \
+	    # install gssntlmssp package from EPEL repository
+      gssntlmssp \
```

Comment:
Indent

Reply:
Fixed the indent issue in GitHub code view. It is apparently properly indented in VSCode and took me at least 2 attempts to fix this invisible whitespace issue that can't be seen in VSCode at all.